src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py

```diff
@@ -101,6 +113,12 @@ def add_layup_info_to_mesh(
         unit_system = material_operators.result_info_provider
 
     layup_provider.inputs.unit_system(unit_system)
+
+    if rst_stream_provider:
```

Comment:
pass the stream of the RST to the lay-up provider if it is available. This check is needed because the old server of the the backend does not support it.